lcviewernoqt/drawitems/lcvspline.cpp

```diff
@@ -9,24 +12,18 @@ LCVSpline::LCVSpline(const lc::entity::Spline_CSPtr spline) : LCVDrawItem(true),
 
 void LCVSpline::draw(LcPainter &painter, const LcDrawOptions &options, const lc::geo::Area &rect) const {
 
-    auto controlPoints = this->controlPoints();
-    if (controlPoints.size()<2) return;
+    auto bezlist = getBeziers();
```

Comment:
We can move this to the constructor so the calculation is done just once, It will cost more memory(We can add a note for that) but that will improve performance.